### ISSUE TITLE
gpt-oss models missing learning rate configs

### ISSUE THREAD

**Issue by xlr8harder** [thinking-machines-lab/tinker-feedback#49]
hyperparam_utils.get_lr is missing configs for gpt-oss models, though the docs indicate you can get them for any  model.  I believe other models are missing as well.

      - openai/gpt-oss-20b
      - openai/gpt-oss-120b

**Comment by bledden** [comment]
Hey @xlr8harder I put a fix up for this at least for gpt-oss - https://github.com/thinking-machines-lab/tinker-cookbook/pull/197
